client/my-sites/people/edit-team-member-form/edit-user-form.jsx

```diff
@@ -50,24 +49,29 @@ class EditUserForm extends Component {
 	}
 
 	getStateObject( props ) {
```

Comment:
Nit: now that we don't use the entire `props` object, perhaps we could restructure it to just assign to whatever we actually need (`user` and `isExternalContributor`)?

Reply:
Yes, absolutely. On my list 👍🏻